docs/tutorials/mongoid-transactions.txt

```diff
@@ -0,0 +1,63 @@
+=========
+Sessions
+=========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Versions 4.0 and higher of the MongoDB server support transactions. You can use sessions with Mongoid in a similar way
+that you would execute a transaction in ActiveRecord as long as the current thread is within a session. Namely, you can
+call a method, ``#with_transaction`` on a model class or on an instance of a model from within a session block and execute
+some operations in a new block. All operations in the inner block will be executed in the context of single transactions. Please
+see the MongoDB Ruby driver documentation for what transaction options are available.
```

Comment:
This needs to reference either the class/method whose options are accepted by name or have a URL to the same. Same for the session doc where this line originated.

Reply:
Modifying the existing session documentation in a way that doesn't relate to transactions is a bit out of scope for this PR, but okay

Reply:
Actually, it looks like the options are documented in detail within the driver itself; I'll add the reference to `Mongo::Client#start_session`, but we'll need to add the documentation there separately